source/_ecosystem/ios/requirements.markdown

```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: "Requirements"
```

Comment:
This seems like a page that can be part of some other page, maybe getting started ?